README.md

```diff
@@ -1085,6 +1088,217 @@ cardNumber.setAltText("Card Number");
 //clear altText
 cardNumber.clearAltText(); 
 ```
+
+## Using Skyflow File Element to upload a file
+
+Like other kinds of data, you can upload binary files in your vault using Skyflow File Element. Follow the steps below to securely upload a file.
+### Step 1: Create a container
+
+First create a container for the form elements using the `container(Skyflow.ContainerType)` method of the Skyflow client as show below:
+
+```javascript
+const container = skyflowClient.container(Skyflow.ContainerType.COLLECT)
+```
+
+### Step 2: Create a file Element
```

Comment:
Include a snippet in this section showing how to create the file element, similar to how you show creating the container in the previous section. Don't just show the object definition.